apollo-router/src/plugins/telemetry/metrics/snapshots/apollo_router__plugins__telemetry__metrics__apollo__test__apollo_metrics_for_subscription_error.snap

```diff
@@ -0,0 +1,61 @@
+---
+source: apollo-router/src/plugins/telemetry/metrics/apollo.rs
+expression: results
+---
+[
+  {
+    "request_id": "[REDACTED]",
+    "stats": {
+      "# -\nsubscription{reviewAdded{body}}": {
+        "stats_with_context": {
+          "context": {
+            "client_name": "test_client",
+            "client_version": "1.0-test",
+            "operation_type": "subscription",
+            "operation_subtype": "subscription-request"
+          },
+          "query_latency_stats": {
+            "latency": {
+              "secs": 0,
+              "nanos": 100000000
+            },
+            "cache_hit": false,
+            "persisted_query_hit": null,
+            "cache_latency": null,
+            "root_error_stats": {
+              "children": {},
+              "errors_count": 0,
+              "requests_with_errors_count": 0
+            },
+            "has_errors": true,
+            "public_cache_ttl_latency": null,
+            "private_cache_ttl_latency": null,
+            "registered_operation": false,
+            "forbidden_operation": false,
+            "without_field_instrumentation": false
+          },
+          "per_type_stat": {}
+        },
+        "referenced_fields_by_type": {
+          "Review": {
+            "field_names": [
+              "body"
+            ],
+            "is_interface": false
+          },
+          "Subscription": {
+            "field_names": [
+              "reviewAdded"
+            ],
+            "is_interface": false
+          }
+        }
+      }
+    },
+    "licensed_operation_count_by_type": {
+      "type": "subscription",
+      "subtype": "subscription-request",
+      "licensed_operation_count": 1
```

Comment:
I might have missunderstood, but i thought this number should be 0 🤔

Reply:
No it's still counting the operation right now, but mark it with `has_errors` to `true`